### DISCUSSION TITLE
Comment faire pour que les SCORMs soient notés?

### DISCUSSION THREAD

**Discussion by Moigaston** [chamilo/chamilo-lms#5383] [General]
Bonjour à tous, 
Dans un parcours la notation s’effectue en pourcentage. Mais comment faire avec les fichiers SCORMs? Et plus exactement comment faire pour que mes apprenants n’aient qu’une seule tentative possible? L’outil à deux flèches qui, si j’ai bien compris, est celui qu’il faut activer pour empêcher plusieurs tentatives, ne marche pas sur ma version de Chamilo 1.11.26. Une solution?
Merci d’avance

**Comment by Moigaston** [comment]
Personne ne sait?

**Comment by LudiscapeXApi** [comment]
Bonjour @Moigaston Moigaston,

Désolé : Pas de note pour les SCORMs.

Dans le contexte des SCORMs, le "mastery score" se réfère au pourcentage minimum requis pour qu'un apprenant soit considéré comme ayant réussi un module de formation. Par exemple, si le "mastery score" est fixé à 80%, cela signifie que l'apprenant doit obtenir au moins 80% des réponses correctes pour être considéré comme ayant réussi le module.

En ce qui concerne la notion de "status completed" et "uncompleted" dans les SCORMs, cela fait référence à l'état dans lequel se trouve un apprenant par rapport à un module donné. Un apprenant est considéré comme ayant atteint le statut "completed" lorsqu'il a terminé avec succès le module en respectant le "mastery score" défini. En revanche, un apprenant est considéré comme "uncompleted" s'il n'a pas encore terminé le module ou s'il n'a pas atteint le "mastery score" requis.

Maintenant, pour ce qui est de votre question, dans le cas de Chamilo 1.11.26, par défaut, le nombre de tentatives est illimité tant que le statut n'est pas "completed". Une fois le statut "completed" atteint, le SCORM reste accessible mais n'évolue plus en termes de statut.

Une solution possible pourrait être de vérifier s'il existe d'autres paramètres dans votre logiciel auteur permettant de limiter le nombre de tentatives des apprenants.

J'espère que cela vous aidera à mieux comprendre le "mastery score" et les concepts de "status completed" et "uncompleted" dans les SCORMs, ainsi qu'à trouver une solution à votre problème de gestion des tentatives dans Chamilo.

Cordialement,

Damien

**Reply by Moigaston** [reply]
Ok, merci Damien!

**Comment by Juliencaporal** [comment]
Bonjour,
Les imports SCORM sur ma plate-forme Chamilo ne remontent pas de score. 

Faut-il procéder à un paramétrage particulier côté Chamilo pour que les scores soient prise en compte ?
Ou bien faut-il considérer dans Chamilo le seul statut de la progression comme le "score" et le mastery score comme la progression devant être accomplie pour valider le module ? 

Le "mastery score" dont vous faites allusion doit être déclaré au moment de l'import, dans le package scorm lui-même, peut être fixé après coup dans Chamilo ?

Un grand merci d'avance

**Comment by LudiscapeXApi** [comment]
Le score est enregistré uniquement si le "mastery score" est dépassé et valide ainsi la progression a 100%.
Cependant il n'est disponible que dans le détail de la tentative (ou en base de données)
